content/harvester/_index.md

```diff
@@ -0,0 +1,28 @@
+# Testing CKAN harvesting of FAIR data points?
+
+Tutorial made on 20/12/2023
+
+We assume that CKAN is properly set up (as documented in this wiki) and running.
+
+After signing in to the CKAN portal, go to CKAN harvest page (e.g. http://localhost:5500/harvest). Click on “Add Harvest Source”.
+
+Fill in the following fields:
+
+- URL of the fair data points. Example: https://fair.healthinformationportal.eu/
+- Title
+- Source type: "FAIR data point harvester"
+- Configuration: { "profile": "fairdatapoint_dcat_ap" }
```

Comment:
This json content could be inside an inline code block, like this `{ "profile": "fairdatapoint_dcat_ap" }`.